### PR TITLE
[Docs] Introduced the "Frameworks" section

### LINKED ISSUE #1213: [Docs] The "Integrating with frameworks" section
TODO

### PR DESCRIPTION
**Pull request by Reinmar**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Docs: Introduced the "Frameworks" section. Closes #1213.

### CODE REVIEW

File: docs/builds/guides/frameworks/angular.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+
+**Note:** You may need to allow external JS in your project's `tsconfig.json` for the builds to work properly:
+
+```json
+"compilerOptions": {
+	"allowJs": true
+}
+```
+
+Now, include the CKEditor module (**TODO WHERE?**):
```
Comment (Reinmar) [inline docs/builds/guides/frameworks/angular.md]: cc @ma2ciek

Comment (ma2ciek) [reply docs/builds/guides/frameworks/angular.md]: Maybe smth like this: `Now, add `CKEditorModule` to your application module imports.`

File: docs/builds/guides/frameworks/overview.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+And passing the `focus: false` option to Boostrap's `modal()` function:
+
+```js
+$( '#modal-container' ).modal( {
+	focus: false
+} );
+```
+
+Check out the demo on https://codepen.io/k0n0pka/pen/yqBXOz.
```
[the remark this answers is not in the document]
Comment (Reinmar) [reply docs/builds/guides/frameworks/overview.md]: @vokiel Do we have such an account already? If not, could you create it?

**Comment by Reinmar** [comment]
I need to update https://ckeditor.com/docs/ckeditor5/latest/builds/guides/faq.html#how-to-use-ckeditor-5-with-frameworks-angular-react-etc too

**Comment by Reinmar** [comment]
I updated the Codepen link and the wording in Angular docs. Ready to merge?

**Pull request merged**